modules/persistent-storage-csi-migration-vsphere.adoc

```diff
@@ -6,82 +6,53 @@
 [id="persistent-storage-csi-migration-sc-vsphere_{context}"]
 = vSphere automatic migration
 
-== New installations of {product-title}
-For new installations of {product-title} 4.13, or later, automatic migration is enabled by default.
-
-[id="upgrading-openshift_{context}"]
-== Upgrading {product-title}
-When upgrading from {product-title} 4.12, or earlier, to 4.13, automatic CSI migration for vSphere only occurs if you opt in.
-
 [IMPORTANT]
 ====
-Carefully review the following consequences before opting in to migration:
-
-* link:https://access.redhat.com/node/7011683[There are known issues that can occur that can cause the migration to fail].
+Migration includes significant consequences:
 
 * Enabling migration cannot be undone.
 
-* Migration can take awhile to complete depending on how many nodes are on the cluster.
-
-* Migration is a disruptive process. The Machine Config Operator (MCO) has to update kubelet, which means degrading each node as it rolls out the new MachineConfig to each machine.
+* Migration can take a while to complete depending on how many nodes are on the cluster.
 ====
```

Comment:
This need not be there anymore in 4.14 docs. Migration is automatic and not something that can be opted-out.

Reply:
@gnufied I know that they can't opt out, but does that mean that they don't need to know about these consequences?

Reply:
OK. Thanks. It's removed.